rgw/v2/tests/s3_swift/configs/test_Mbuckets_with_Nobjects_failed.yaml

```diff
@@ -0,0 +1,21 @@
+# upload type: non multipart
+# script: test_Mbuckets_with_Nobjects.py
+config:
+  user_count: 1
+  bucket_count: 2
+  objects_count: 100
+  objects_size_range:
+    min: 5
+    max: 15
+  test_ops:
+    create_bucket: true
+    create_object: true
+    upload_type: failed_upload
```

Comment:
@TejasC88 , can we update the type to read_only_update, for better usability.

Reply:
Done